src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -337,6 +337,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <EmccArgs>&quot;$(LlvmObject)&quot; -c -o &quot;$(NativeObject)&quot; -s DISABLE_EXCEPTION_CATCHING=0</EmccArgs>
       <EmccArgs Condition="'$(NativeDebugSymbols)' != 'true'">$(EmccArgs) -O2</EmccArgs>
       <EmccArgs Condition="'$(NativeDebugSymbols)' == 'true'">$(EmccArgs) -g3</EmccArgs>
+      <EmccArgs Condition="'$(EmccShellFile)' != ''">$(EmccArgs) --shell-file $(EmccShellFile)</EmccArgs>    
```

Comment:
This is the compile-time args section, `--shell-file` is a link-time flag, so it would have to be added [below](https://github.com/dotnet/runtimelab/blob/50151a1fd5cbf06e024338895ce9e18b8278058e/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L418-L422).

Additionally, if we don't want too much Emscripten specifics to leak out, `EmccShellFile` -> `HtmlShellFile`?